internal/infrastructure/event/consumer.go

```diff
@@ -146,10 +149,11 @@ func processConsumedMessage(schemas schema.TopicSchema, msg *kafka.Message, hand
 	if internalTopic == "" {
 		return fmt.Errorf("Topic mapping not found for: %s", *msg.TopicPartition.Topic)
 	}
-	log.Info().
-		Str("Topic name", *msg.TopicPartition.Topic).
-		Str("Requested topic name", internalTopic).
-		Msg("Topic mapping")
+	slog.Info(
+		"Topic mapping",
+		slog.String("Topic name", *msg.TopicPartition.Topic),
```

Comment:
Spaces in struct logging keys create awkward messages like `Topic mapping "Topic name"=topic ...`. I suggest that we use keys that do no need quotes.

Reply:
Fixed.

Reply:
the current state include the suggested change; resolving the comment.